deploy/all-in-one/README.md

```diff
@@ -0,0 +1,31 @@
+# Deploying with the All-In-One Docker Container
+
+The All-in-One container sets up the following components in a single container:
+1. The Stratos UI frontend and backend
+2. An UAA instance
+
+## Requirements:
+
+You will need to have installed both Docker, see:
+
+* [Docker](https://docs.docker.com/engine/installation/)
+
+## Building the container:
+
+To build the container, change directory to the root of the project and open a command prompt:
```

Comment:
They don't need to build the container do they? Doesn't it use the one we publish?